pkg/inject/report.go

```diff
@@ -77,6 +78,7 @@ func newReport(conf *ResourceConfig) *Report {
 		report.HostNetwork = conf.pod.spec.HostNetwork
 		report.Sidecar = healthcheck.HasExistingSidecars(conf.pod.spec)
 		report.UDP = checkUDPPorts(conf.pod.spec)
+		report.TracingEnabled = conf.pod.meta.Annotations[k8s.ProxyTraceCollectorSvcAddr] != "" || conf.nsAnnotations[k8s.ProxyTraceCollectorSvcAddr] != ""
```

Comment:
Feel free to ignore for now, but just a forward looking question:

Are reports always based just on the annotations?  If we, in the future, support setting default tracing values in the linkerd-config, is it possible to report that tracing is enabled even when it's not on the annotations?

Reply:
Yes, if down the road, we have other ways to enable tracing, we will need to account for that here.